api/controllers/album.js

```diff
@@ -38,3 +75,58 @@ export async function byAlbumID(req, res) {
     }
   }
 }
+/**
+ * function: getAlbumQuery
+ * creates query and parameters for an album query to the database
+ *
+ * @param {String} name - string for fuzzy search
+ * @param {String} tag - tag for fuzzy search
+ * @param {Int} parentID - parentID for exact search
+ * @param {String} date - updated after given date
+ * @param {String} keyword - keyword for fuzzy search
+ * @param {Int} albumID - albumID for exact search
+ *
+ * @return {Object[String, Array]} = template statement, parameters for template statement
+ */
+function getAlbumQuery(name, tag, parentID, date, keyword, albumID) {
```

Comment:
change the input to an object so order can be ignored. 
```
function getAlbumQuery({ name, tag, parentID, date, keyword, albumID }) {
```

you would call it like 
`getAlbumQuery({name: 'foo', tag: 'bar' }); // input object can any of the props.`